post release 1.5.1 (#329)
crystal-lang/crystal-website: _releases/2022-09-06-1.5.1-released.md

---
title: Crystal 1.5.1 is released!
version: 1.5.1
summary:
thumbnail: +
author:
---

We are releasing the first patch release of the [1.5](https://crystal-lang.org/2022/07/06/1.5.0-released.html) series.

Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.5.1) and our official distribution channels.
See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.

## Stats

In this patch release we included [10 PRs since the 1.5.0 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.5.1) by 4 contributors. We thank all the effort put into improving the language! ❤️

## Remarkable changes

> It was fixed a regression that was introduced in 1.5.0. This prevented the typechecker to correctly infer the type of a union of a splat of types (something of the form `Union(*T)`). To illustrate, the following code typechecks in 1.4.1 but not in 1.5.0.

Reviewer: ```suggestion
A regression introduced in 1.5.0 prevented the typechecker to correctly infer the type of a union of a splat of types (something of the form `Union(*T)`). This is now fixed.

To illustrate, the following code typechecks in 1.4.1 but not in 1.5.0.
```
